contracts/gas-service/AxelarGasService.sol

```diff
@@ -19,8 +19,15 @@ contract AxelarGasService is Upgradable, IAxelarGasService {
     using SafeTokenTransferFrom for IERC20;
     using SafeNativeTransfer for address payable;
 
+    // keccak256('AxelarGasService.Slot')
```

Comment:
In my personal opinion those `- 1` tricks don't help anything. `keccak()` and `keccak() - 1` have equal probability of collision with something else. But if you have a strong preference we could add it and follow an established pattern for this

Reply:
Added `- 1` to the SDK PR

Reply:
Some of our contracts have computed storage slots based on user provided values (albeit prefixed). So the concern would be that there's a collision if we aren't careful about prefixing (including if contracts are inherited). The `-1` for predefined slots could help avoid that. Unlikely to happen, but it could be concerning if it did.